src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -667,5 +667,231 @@ public void SarifTransformerTests_ToVersionOne_OneRunWithInvocationAndNotificati
 
             VerifyCurrentToVersionOneTransformation(V2LogText, V1LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_NotificationExceptionWithStack()
+        {
+            const string V2LogText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""invocations"": [
+        {
+          ""toolNotifications"": [
+            {
+              ""id"": ""CTN0001"",
+              ""message"": {
+                ""text"": ""Unhandled exception.""
+              },
+              ""level"": ""error"",
+              ""exception"": {
+                ""kind"": ""ExecutionEngine.RuleFailureException"",
+                ""message"": ""Unhandled exception during rule evaluation."",
+                ""stack"": {
+                  ""message"": {
+                    ""text"": ""This is the stack messasge.""
+                  },
+                  ""frames"": [
+                    {
+                      ""location"": {
+                        ""physicalLocation"": {
+                          ""fileLocation"": {
+                            ""uri"": ""file:///C:/src/main.cs""
+                          },
+                          ""region"": {
+                            ""startLine"": 15,
+                            ""startColumn"": 9
```

Comment:
@michaelcfanning @EasyRhin0 

Chris, you have no action for now, but be aware:

At TC #&#xFEFF;18 on May 30th, we approved changes to the default values of missing region properties. In particular, if `endColumn` is missing, in v1 that meant `endColumn` == `startColumn` (an insertion point). But in v2, `endColumn` defaults to “one character past EOL (excluding the newline sequence”).

The reason for this change is that one of the TC members argued persuasively that the most natural interpretation of the region `{ "startLine": 3 }` is "The whole of line 3" and not "An insertion point at the beginning of line 3."

So when you get around to making the schema changes for oasis-tcs/sarif-spec#93, you'll need to adjust the transformations as well.

Reply:
OK thanks